sources/httpUtils.ts

```diff
@@ -27,28 +42,41 @@ export async function fetchUrlStream(url: string, options: RequestOptions = {})
   });
 }
 
-export async function fetchAsBuffer(url: string, options?: RequestOptions) {
+export async function fetchAsBuffer(url: string, hash?: string, options?: RequestOptions) {
```

Comment:
nit: I'd put the hash as part of the option bag

Reply:
I didn't do it because `options` is passed to the Node.js `https.get` method, and I thought it could be problematic if it clashes in the future with one of Node.js' supported options – and if I'm being honest, I was feeling too lazy to update the types.

Reply:
I've refactored that away, I now rely on semver to parse the build info and removed the additional argument.